demo/README.md

```diff
@@ -65,7 +67,7 @@ The Bookstore, Bookbuyer, and Bookthief apps have simple web UI visualizing the
   - To see Grafana run `./scripts/port-forward-grafana.sh` and open [http://localhost:3000/](http://localhost:3000/) - default username and password for Grafana is `admin`/`admin`
   - OSM controller has a simple debugging web endpoint - run `./scripts/port-forward-osm-debug.sh` and open [http://localhost:9091/debug](http://localhost:9091/debug)
 
-To expose web UI ports of all components of the service mesh the local workstation use the following helper script: `/scripts/port-forward-all.sh`
+To expose web UI ports of all components of the service mesh the local workstation use the following helper script: `/scripts/port-forward-all.sh`  [Note: this script requires GNU parallel]
```

Comment:
I removed the dependency for `parallel` in https://github.com/openservicemesh/osm/pull/1430. Could you remove the note.